src/Scaffold/Console/controller/controller.stub

```diff
@@ -8,14 +8,29 @@ use Backend\Classes\Controller;
  */
 class {{studly_name}} extends Controller
 {
+    /**
+     * @var array Behaviors that are implemented by this controller.
+     */
     public $implement = [
         'Backend.Behaviors.FormController',
         'Backend.Behaviors.ListController'
     ];
 
+    /**
+     * @var string Configuration file for the `FormController` behavior.
+     */
     public $formConfig = 'config_form.yaml';
+
+    /**
+     * @var string Configuration file for the `ListController` behavior.
+     */
     public $listConfig = 'config_list.yaml';
 
+    /**
+     * @var array Permissions required to access this controller's actions.
+     */
+    public $requiredPermissions = ['{{lower_author}}.{{lower_plugin}}.{{lower_name}}'];
```

Comment:
This shouldn't be included since that would require them to register a permission in the form of author.plugin.controller which isn't obvious to newcomers and would block access to the controller to anyone not a superadmin.

Reply:
But without permission anybody can access this and new people dont know that and that makes it as security issue no? And maybe if we add this we should add default permission for plugin that is like `plugin.name.*`.

Reply:
Not anybody, only authenticated backend users. I wouldn't be opposed to a wildcard necessarily, but it still presents a problem because default scaffolded plugins don't provide any permissions.  

If we also make a change to the plugin scaffolding to have the plugins register a default permission of `author.plugin.manage_plugin` and then check for that here that should be fine.

Reply:
Check for a wildcard that is

Reply:
@LukeTowers i think for now i deleted(or you can) this to merge this PR and then later i make another PR with changes to permissions.

Reply:
@LukeTowers removed, later i make separate PR for it, PR should be free to merge.